sys-apps/systemd/files/98-fleet-org.freedesktop.systemd1.rules

```diff
@@ -0,0 +1,6 @@
+polkit.addRule(function(action, subject) {
```

Comment:
For clarity, from a distro packager's point of view the ideal package just ships upstream code and doesn't need distro specific hacks/patches. So given that fleet depends on systemd but not the other way around fleet related policy would be weird for systemd itself to ship.